pages/connections/farmlands.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <main>
+    <div
+      class="bg-bottom bg-cover bg-[#174B33] bg-blend-overlay bg-[url('~/assets/images/countryside.jpg')]"
```

Comment:
Is `#174B33` not a color in our color scheme?

Reply:
Nah it isn't - maybe it should be added if we plan to keep using this green for farmlands related docs